swap_meet/vendor.py

```diff
@@ -1,2 +1,82 @@
 class Vendor:
-    pass
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
This works fine, but I would tend to invert the sense of the comparison to use it as a guard clause, allowing us to avoid indenting the main part of the method, as well as the lonely return at the end. Something like
```python
        if item not in self.inventory:
            return False
            
        self.inventory.remove(item)
        return item
```

Al alternative approach would be to try to remove the item without checking, and catch the `ValueError` that would be raised if the item wasn't in the list, returning False in the error handler.